packages/redux/src/users/closets/actions/__tests__/resetUserClosetSummaries.test.ts

```diff
@@ -0,0 +1,23 @@
+import * as actionTypes from '../../actionTypes.js';
```

Comment:
It seems this file should be called `resetUserCloset.test.ts`, right?

Reply:
Yes, correct. Fixed.